### PR TITLE
updates for DFG#440

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: src/DispatchPackedConversions.jl [hunk 1]
```diff
@@ -45,7 +45,7 @@ function convert(
   ccw = prepgenericconvolution(DFG.DFGVariable[], usrfnc, multihypo=mhcat, nullhypo=nh)
   ccw.certainhypo = d.certainhypo
 
-  ret = FunctionNodeData{CommonConvWrapper{typeof(usrfnc)}}(d.fncargvID, d.eliminated, d.potentialused, d.edgeIDs, Symbol(d.frommodule), ccw, d.multihypo, d.certainhypo)
+  ret = FunctionNodeData{CommonConvWrapper{typeof(usrfnc)}}( d.eliminated, d.potentialused, d.edgeIDs, ccw, d.multihypo, d.certainhypo, d.solveInProgress)
```
Comment (Affie) [inline src/DispatchPackedConversions.jl:48]: @dehann should solveInProgress be reset to 0 or also unpacked?

File: src/DispatchPackedConversions.jl [hunk 2]
```diff
@@ -20,9 +20,9 @@ end
 
 function convert(::Type{PackedFunctionNodeData{P}}, d::FunctionNodeData{T}) where {P <: PackedInferenceType, T <: FactorOperationalMemory}
   # mhstr = packmultihypo(d.fnc)  # this is where certainhypo error occurs
-  return PackedFunctionNodeData(d.fncargvID, d.eliminated, d.potentialused, d.edgeIDs,
-          string(d.frommodule), convert(P, d.fnc.usrfnc!),
-          d.multihypo, d.fnc.certainhypo )  # extract two values from ccw for storage -- ccw thrown away
+  return PackedFunctionNodeData(d.eliminated, d.potentialused, d.edgeIDs,
+          convert(P, d.fnc.usrfnc!),
+          d.multihypo, d.fnc.certainhypo, d.solveInProgress)  # extract two values from ccw for storage -- ccw thrown away
```
Comment (Affie) [inline src/DispatchPackedConversions.jl:25]: Also for packing

File: src/DispatchPackedConversions.jl (hunk 1 above)
Comment (dehann) [reply src/DispatchPackedConversions.jl:48]: should be unpacked

Comment (dehann) [reply src/DispatchPackedConversions.jl:48]: idea is to get concurrency through the DB, so all concurrency flags need to be maintained and should not be reset locally.

Comment (dehann) [reply src/DispatchPackedConversions.jl:48]: "maintained" being a polling (or pull) model, not a notification/push model.

File: src/DispatchPackedConversions.jl (hunk 2 above)
Comment (dehann) [reply src/DispatchPackedConversions.jl:25]: thanks!

File: src/FactorGraph.jl [hunk 3]
```diff
@@ -646,7 +646,7 @@ function getDefaultFactorData(
   ccw = prepgenericconvolution(Xi, usrfnc, multihypo=mhcat, nullhypo=nh, threadmodel=threadmodel)
 
   # and the factor data itself
-  data_ccw = FunctionNodeData{CommonConvWrapper{T}}(Int[], false, false, Int[], Symbol(T.name.module), ccw, multihypo, ccw.certainhypo)
+  data_ccw = FunctionNodeData{CommonConvWrapper{T}}(false, false, Int[], ccw, multihypo, ccw.certainhypo, 0)
```
Comment (dehann) [inline src/FactorGraph.jl:649]: breadcrum for solveInProgress

Comment (Affie) [reply src/FactorGraph.jl:649]: So this is a new object, should sholveInProgress be a parameter?

**Pull request merged**